stl/inc/iterator

```diff
@@ -164,13 +170,14 @@ public:
 
 protected:

```

Comment:
Cleanup: This should be `private`. The affected members are all `_Ugly` and we don't have any STL machinery inheriting from `istream_iterator`. (This will affect the mangled name of `_Getval` but that's an ABI-safe change.)